libraries/AP_RCProtocol/AP_RCProtocol.h

```diff
@@ -250,6 +250,17 @@ class AP_RCProtocol {
     void handle_radio_rc_channels(const mavlink_radio_rc_channels_t* packet);
 #endif
 
+    // some backends have a struct LinkStatus and a field _link_status, avoid name clash
+    struct RcLinkStatus {
+        int16_t rssi = -1;
+        int16_t link_quality = -1;
+        int8_t rssi_dbm = -1;
+        int8_t snr = INT8_MIN;
+    };
+    volatile struct RcLinkStatus _rc_link_status;
```

Comment:
```suggestion
    volatile struct RCLinkStatus _rc_link_status;
```

Why `volatile`?

Reply:
this I didn't understand too ... so I was just closily following what is done for the CRSF class 
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_RCProtocol/AP_RCProtocol_CRSF.h#L310-L314
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_RCProtocol/AP_RCProtocol_CRSF.h#L361
assuming that someone had a good reason which is above my paygrade

pl say clear "yes volatile" or "no voloatile" and I will do

Reply:
has the "volatile" thing been sorted out? It should stay?